pkgs/applications/misc/tint2/default.nix

```diff
@@ -1,30 +1,41 @@
 { stdenv, fetchurl, pkgconfig, cmake, pango, cairo, glib, imlib2, libXinerama
 , libXrender, libXcomposite, libXdamage, libX11, libXrandr, gtk, libpthreadstubs
-, libXdmcp
+, libXdmcp, librsvg, fetchgit
 }:
 
 stdenv.mkDerivation rec {
   name = "tint2-${version}";
-  version = "0.11";
+  version = "0.12";
 
-  src = fetchurl {
-    url = "http://tint2.googlecode.com/files/${name}.tar.bz2";
-    sha256 = "07a74ag7lhc6706z34zvqj2ikyyl7wnzisfxpld67ljpc1m6w47y";
+  src = fetchgit {
+    url = "https://gitlab.com/o9000/tint2.git";
+    rev = "107bfc85a3955a8b88ff027d2f6cb4cb7e943cb0";
```

Comment:
This is the revision for version 0.12.7, not 0.12: https://gitlab.com/o9000/tint2/commit/107bfc85a3955a8b88ff027d2f6cb4cb7e943cb0

To make this cleaer you should just use a tag here which for this package corresponds to the version, so: `rev = version;`